blackbox/docs/sql/optimize.txt

```diff
@@ -0,0 +1,95 @@
+.. highlight:: psql
+.. _optimize:
+
+========
+Optimize
+========
+
+In Crate every table (or if partitioned every partition) consists of segments.
+When inserting/deleting/updating data new segments are created and there is the
+danger of having the number of segments grow too big. Each segment consumes file
```

Comment:
wouldn't write it like that (danger of...) also would be better to explain why there are multiple segments (append-only, faster writes, but less segments == faster reads)